Source/Core/VideoBackends/OGL/PostProcessing.cpp

```diff
@@ -37,12 +37,18 @@ OpenGLPostProcessing::~OpenGLPostProcessing()
 }
 
 void OpenGLPostProcessing::BlitFromTexture(TargetRectangle src, TargetRectangle dst,
-                                           int src_texture, int src_width, int src_height,
-                                           int layer)
+                                           CropRectangle crop_rc, int src_texture, int src_width,
+                                           int src_height, int layer)
 {
   ApplyShader();
 
+  GLboolean old_scissor_enabled = glIsEnabled(GL_SCISSOR_TEST);
```

Comment:
Okay, that makes sense to me. I added `glDisable(GL_SCISSOR_TEST)` in SwapImpl, and `glScissor(0, 0, GetTargetWidth(), GetTargetHeight());` right after the scissor test is enabled in RestoreAPIState.